### PR TITLE
Task 1: Should contain react static view of recipe list

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0",
+    "react"
+  ]
+}
```
Comment (pjazdzewski1990) [inline .babelrc:7]: Should It be committed into the repo? (Not sure about that either)

File: src/components/RecipeBox.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+				<div className="recipeDesc">
+					<h3>{this.props.recipeName}</h3>
+					<p>{this.props.description}</p>
+				</div>
+			</div>
+		);
+	}
+}
+
+export default RecipeBox;
```
Comment (pjazdzewski1990) [inline src/components/RecipeBox.js]: Q: If you put `export` at class declaration is another export here needed?

File: src/components/RecipeBox.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+export class RecipeBox extends Component {
+	render() {
+		return (
+			<div className="recipeBox">
```
Comment (pjazdzewski1990) [inline src/components/RecipeBox.js]: Q: What is the encouraged convention for css? `cssClassName` or `css-class-name`?

File: src/containers/App.js [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
+              {name: 'Bar4', description: 'Baaar', image: require('../images/tortilla.jpg')},
+              {name: 'Baz4', description: 'Baz', image: require('../images/cheeserolls.jpg')}];
+
+class App extends React.Component {
+	render() {
+		return(
+			<div>
+                            <div id="mainImg">
+				   <img src={require('../images/banner.jpg')} />
+                            </div>
```
Comment (pjazdzewski1990) [inline src/containers/App.js]: I think something went from with formatting here

File: .babelrc (hunk 1 above)
Comment (burczu) [reply .babelrc:7]: yes, this should be committed to the repo

File: src/components/RecipeBox.js (hunk 2 above)
Comment (burczu) [reply src/components/RecipeBox.js]: no, here should be only one export

File: src/components/RecipeBox.js (hunk 3 above)
Comment (burczu) [reply src/components/RecipeBox.js]: basically more common approach is `css-class-name` but it depends on decision made by the team

File: src/components/RecipesList.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+require('normalize.css/normalize.css');
+require('styles/App.css');
+
+import React, { Component } from 'react';
+import RecipeBox from './RecipeBox.js';
+
+export class RecipesList extends Component {
+	render() {
+		let recipesBoxes = this.props.data.map(recipe => {
```
Comment (burczu) [inline src/components/RecipesList.js]: `const` is preferred if you don't change the variable later, additionally - this code could be extracted to the separated method of the class

File: src/containers/App.js [hunk 6]
```diff
@@ -1,30 +1,30 @@
...
+              {name: 'Foo4', description: 'Foooooooo', image: require('../images/wings.jpg')},
+              {name: 'Bar4', description: 'Baaar', image: require('../images/tortilla.jpg')},
+              {name: 'Baz4', description: 'Baz', image: require('../images/cheeserolls.jpg')}];
+
+class App extends React.Component {
+	render() {
+		return(
+			<div>
+                            <div id="mainImg">
+				   <img src={require('../images/banner.jpg')} />
```
Comment (burczu) [inline src/containers/App.js]: it is better to import images (using `import` keyword) at the beginning of the file (perhaps specific webpack loader will be necessary)

File: src/components/RecipeBox.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+				<img src={this.props.image}></img>
```
Comment (pjazdzewski1990) [inline src/components/RecipeBox.js:7]: Today we discussed the need for https://facebook.github.io/react/docs/reusable-components.html#prop-validation

File: .babelrc (hunk 1 above)
Comment (ghost) [reply .babelrc:7]: ok, I leave it as is

File: src/components/RecipeBox.js (hunk 2 above)
Comment (ghost) [reply src/components/RecipeBox.js]: removed `export` before class declaration

File: src/components/RecipeBox.js (hunk 3 above)
Comment (ghost) [reply src/components/RecipeBox.js]: leaving as is

File: src/containers/App.js (hunk 4 above)
Comment (ghost) [reply src/containers/App.js]: changing formatting to something more aesthetic :)

File: src/components/RecipesList.js (hunk 5 above)
Comment (ghost) [reply src/components/RecipesList.js]: done

File: src/containers/App.js (hunk 6 above)
Comment (ghost) [reply src/containers/App.js]: currently when I try to `import` I receive lots of errors

**Pull request merged**